### PR TITLE
(DOCSP-23216): v1.1.3 Release Notes & source constant

### PR DESCRIPTION
**Pull request by melissamahoney-mongodb**
[JIRA](https://jira.mongodb.org/browse/DOCSP-23216)
[Staging](https://docs-mongodbcom-staging.corp.mongodb.com/atlas-cli/docsworker-xlarge/DOCSP-23216/atlas-cli-changelog/)
[Builds clean](https://workerpool-boxgs.mongodbstitch.com/pages/job.html?collName=queue&jobId=62a788577cf5397525c11b62) (aside from known completion errors)

### CODE REVIEW

File: source/atlas-cli-changelog.txt [hunk 1]
```diff
@@ -12,6 +12,26 @@
...
+.. _atlas-cli-1.1.3:
+
+{+atlas-cli+} 1.1.3
+-------------------
+
+Released 13 June 2022
+
+- Improves the :ref:`atlas-quickstart` command:
+
+  - Prompts users to log in if they are not already logged in.
```
Comment (JuliaMongo) [inline source/atlas-cli-changelog.txt:24]: you

Comment (JuliaMongo) [inline source/atlas-cli-changelog.txt:24]: if you aren't

**Review by JuliaMongo — APPROVED** [review]
LGTM with a few small comments. Sorry for the delay!

**Review by blva — APPROVED** [review]
LGTM, thanks for the changes!

**Pull request merged**